README.md

```diff
@@ -28,8 +28,9 @@ The provided boards are **NOT** supported by the .NET **nanoFramework** team and
 | [TI_CC1352P1_LAUNCHXL_868](TI_SimpleLink/TI_CC1352P1_LAUNCHXL)      | [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/TI_CC1352P1_LAUNCHXL_868/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/TI_CC1352P1_LAUNCHXL_868/latest/)     | TI CC1352     | 48MHz     | 352kB   | 80kB    |
 | [TI_CC1352P1_LAUNCHXL](TI_SimpleLink/TI_CC1352P1_LAUNCHXL_915)      | [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/TI_CC1352P1_LAUNCHXL_915/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/TI_CC1352P1_LAUNCHXL_915/latest/)     | TI CC1352     | 48MHz     | 352kB   | 80kB    |
-| [PybStick Std 26](PybStick2x) | [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/PybStick2x/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/PybStick2x/latest/) | STM32F411RE   | 100 MHz   | 512 kB  | 128 kB  |
-| NETDUINO3_WIFI |  [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/NETDUINO3_WIFI/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/NETDUINO3_WIFI/latest/) | STM32F427VI   | 168 MHz   | 1408kB  | 164kB  |
+| [PybStick Std 26](ChibiOS/PybStick2x) | [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/PybStick2x/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/PybStick2x/latest/) | STM32F411RE   | 100 MHz   | 512 kB  | 128 kB  |
+| [NETDUINO3_WIFI](ChibiOS/NETDUINO3_WIFI) |  [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/NETDUINO3_WIFI/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/NETDUINO3_WIFI/latest/) | STM32F427VI   | 168 MHz   | 1408kB  | 164kB  |
+| [BrainPad2](ChibiOS/BrainPad2) |  [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/BrainPad2/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/BrainPad2/latest/) | STM32F401RE   | 84 MHz   | 512 kB  | 96 kB  |
```

Comment:
Please extract this to another PR.

Reply:
Probably easier to do it the other way around 😁